glusterd2/commands/volumes/volfiles.go

```diff
@@ -33,3 +37,38 @@ func volfilesListHandler(w http.ResponseWriter, r *http.Request) {
 	}
 	restutils.SendHTTPResponse(ctx, w, http.StatusOK, volfiles)
 }
+
+func volfilesGetHandler(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+
+	volfiles, err := volgen.GetVolfiles()
+
+	if err != nil {
+		restutils.SendHTTPError(ctx, w, http.StatusInternalServerError, "unable to get list of volfiles")
+		return
+	}
+
+	restutils.SendHTTPResponse(ctx, w, http.StatusOK, volfiles)
+}
+
+func volfileGetHandler(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	volfile := mux.Vars(r)["volfileid"]
+
+	//remove extra slash in last ( as StrictSlash is false in mux.Route)
+	volfile = strings.TrimSuffix(volfile, "/")
+	volfiles, err := volgen.GetVolfile(volfile)
```

Comment:
This is a single volfile. Name this variable as volfile.

Reply:
renamed variable